src/controllers/FacetQueryController.ts

```diff
@@ -41,7 +41,7 @@ export class FacetQueryController {
   }
 
   /**
-   * Compute the filter expression that the facet need to output for the query
```

Comment:
In general, use the third person singular for action verbs when documenting methods:

> Generates the filter expression that the facet needs to output for the query

For the sake of completion, we could also document the return value:

> @returns {string} The generated filter expression.